packages/launchpad/src/migration/OptOutModalStep2.vue

```diff
@@ -2,6 +2,7 @@
   <StandardModal
     :model-value="true"
     :title="t('migration.renameAuto.modals.title')"
+    @close="emit('cancel')"
```

Comment:
Do we need to pass this here since the cancel button emit's 'cancel' as well?

Reply:
hmm maybe i'm misunderstanding but isn't this so when the modal emits a close event we actually close the modal?

Reply:
That should be handled by the StandardModel here: https://github.com/cypress-io/cypress/blob/533572ed9352d20ee1b56043e6c3169031a1aeb9/packages/frontend-shared/src/components/StandardModal.vue#L90

Are you testing specifically for outside click? We'd want to pass the clickOutside prop as true to enable.

Reply:
@emilyrohrbough the standard modal component does not actually handle its own visibility state.
It delegates its value to the parent component via a v-model system (2 way binding in vue).

For it to work one has to: 
Have a `showModal` reactive variable in the parent component.
Use it as the v-model prop of the StandardModal. 

Since it would not delete the parent component if I used it, I preferred using my own visibility system one level up. In another comment I explain a simpler way to do what is done here without changing the standardmodal.